### DISCUSSION TITLE
Duda del enunciado: Características

### DISCUSSION THREAD

**Discussion by Ckanerloy** [dds-jv/foro#220] [TPA - 2021] [answered]
Hola, buenas tardes!. Tenemos una duda acerca de item 2 de requerimientos generales ```Se debe permitir que un administrador agregue posibles características de mascotas con facilidad.```
Queremos saber si el administrador debe crear un nueva característica en el momento de utilizar el sistema o debe tener la posibilidad de seleccionar alguna característica de las ya existentes.

**Comment by gastonprieto** [comment, marked as answer]
Buenas @Ckanerloy !

El administrador es quien las da de alta en el sistema.

**Reply by tomaslcarballo** [reply]
Perdón que me meto pero quiero ver si entendí bien, es decir que ya estan creadas las caracteristicas y él solo las "habilita"?

**Reply by flbulgarelli** [reply]
¡Hola!

Depende qué entiendas por _ya están creadas_. Por un lado, sí, los componentes necesarios para poder crear las características tienen que existir previamente en el sistema, y estos componentes permitirán (y limitarán) hasta cierto punto definir ciertas características en tiempo de ejecución. 

Pero por otro lado, no, no es que todas las características existen, predefinidas, en el sistema, sino que vos vas a utilizar componentes basales para que definir las tuyas propias. Por ejemplo, es válido que en el sistema existan componentes para decir "característica obligatoria numérica que puede valer entre 0 y 20" y que le admin lo utilice para definir la edad, pero no es váido que el sistema tenga precargada la característica "edad" y le admin simplemente la active. 

¡Saludos!

**Reply by gastonprieto** [reply]
Buenas @Ckanerloy y @tomaslcarballo les sirvio lo que les comentamos?

Saludos!

**Reply by Ckanerloy** [reply]
Sisi,muchas gracias!!

**Reply by tomaslcarballo** [reply]
Sisi, gracias!
